resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,385 @@
+      - Departures
+      summary: Inject a fake NCTS Departure Message
+      description: >
+        Inject a message type into the test NCTS database for a given Movement Departure.
+      operationId: postFakeNCTSDepartureMessage
+      parameters:
+        - $ref: '#/components/parameters/contentTypeHeader'
+        - in: path
+          name: departureId
+          description: The ID specifying the Departure
+          required: true
+          style: simple
+          schema:
+            type: string
+            example: '1'
+      requestBody:
+        description: JSON with message type
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/TestCustomsTransitsMovementsDeparturesMessagesRequest'
+              - description: JSON with message type
+                example:
+                  message:
+                    messageType: IEXXX
+            example:
+              message:
+                messageType: IEXXX
+        required: true
+      responses:
+        '201':
+          description: When a test message has been successfully injected.
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/TestCustomsTransitsMovementsDeparturesMessagesResponse'
+                - description: JSON Payload
+                  example:
+                    _links:
+                      self:
+                        href: /customs/transits/movements/departures/1/messages/2
+                      departure:
+                        href: /customs/transits/movements/departures/1
+                    departureId: '1'
+                    messageId: '2'
+                    messageType: IEXXX
+                    body: <CCXXX>...</CCXXX>
+              example:
+                _links:
+                  self:
+                    href: /customs/transits/movements/departures/1/messages/2
+                  departure:
+                    href: /customs/transits/movements/departures/1
+                departureId: '1'
+                messageId: '2'
+                messageType: IEXXX
+                body: <CCXXX>...</CCXXX>
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/errorResponse'
+                  - description: The supplied payload was invalid.
+                    example:
+                      statusCode: 400
+              example:
+                statusCode: 400
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/errorResponse'
+                  - description: When the Arrival Movement does not exist, has been archived or is not available to the EORI number.

```

Comment:
Is arrival correct here?

Supplied departure not found or does not exist or has been archived or is not available to the EORI number.